### PR TITLE
DM-22039: Rename dax_ppdb to dax_apdb (ap_pipe)

### PR DESCRIPTION
**Pull request by andy-slac**
Renames all PPDB names into APDB.

### CODE REVIEW

File: doc/lsst.ap.pipe/apdb.rst [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 
-.. |make_ppdb| replace:: :doc:`make_ppdb.py <scripts/make_ppdb.py>`
+.. |make_apdb| replace:: :doc:`make_apdb.py <scripts/make_apdb.py>`
 
 .. |ap_pipe| replace:: :command:`ap_pipe.py`
 
-|ap_pipe| needs an existing Prompt Products Database (PPDB) in which to store its results.
-Such a database will be provided externally during operations, but developers can run |make_ppdb| to set up their own database for testing.
-This page provides an overview of how to use |make_ppdb|.
+|ap_pipe| needs an existing Prompt Products Database (APDB) in which to store its results.
```
Comment (kfindeisen) [inline doc/lsst.ap.pipe/apdb.rst]: Alert Production Database

File: doc/lsst.ap.pipe/pipeline-tutorial.rst [hunk 2]
```diff
@@ -112,7 +112,7 @@ The result from running ``ap_pipe`` should look something like
 
 .. code-block:: none
 
-   ppdb/
+   apdb/
       association.db   <--- the Prompt Products Database with DIAObjects
```
Comment (kfindeisen) [inline doc/lsst.ap.pipe/pipeline-tutorial.rst]: Alert Production Database

File: python/lsst/ap/pipe/make_apdb.py [hunk 3]
```diff
@@ -87,8 +87,8 @@ def parse_args(self, args=None, namespace=None):
         return namespace
 
 
-def makePpdb(args=None):
-    """Create a PPDB according to a config.
+def makeApdb(args=None):
+    """Create a APDB according to a config.
```
Comment (kfindeisen) [inline python/lsst/ap/pipe/make_apdb.py]: a -> an

**Comment by andy-slac** [comment]
Thanks, I thought I replaced all "Prompt Product Database" but apparently missed some (or all) in this package. I ran another check and fixed them all.

**Pull request merged**